BUILDING.md

```diff
@@ -385,3 +385,21 @@ and [user guide](https://openssl.org/docs/fips/UserGuide-2.0.pdf).
    `/usr/local/ssl/fips-2.0`
 8. Build Node.js with `make -j`
 9. Verify with `node -p "process.versions.openssl"` (for example `1.0.2a-fips`)
+
+## Building Node.js with external core modules
+
+In general those core modules are only located in `lib/` folder.  Node.js can
```

Comment:
The word "those" here looks a bit unnatural since the core modules are not mentioned before this section, and the fact that they are in the `lib/` folder is not entirely relevant here since the internal modules are not exposed to users. I would suggest something simpler like:

```
It is possible to specify one or more JavaScript text files to be bundled in the binary as builtin modules
when building Node.js.
```